app/views/challenge/shared/_challenge_form_confirm_delete.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for(@collection, :url => 
+  if @collection.prompt_meme?
+    collection_prompt_meme_path(@collection)
+  else
+    collection_gift_exchange_path(@collection)
+  end,
+  :html => {:method => :delete}) do |f| %>
```

Comment:
Could you add spaces inside the braces here, e.g. `:html => { :method => :delete }`?